amp/WEB-INF/src/org/digijava/module/aim/annotations/activityversioning/ActivityVersioningService.java

```diff
@@ -0,0 +1,377 @@
+package org.digijava.module.aim.annotations.activityversioning;
```

Comment:
also to avoid introducing a new file we could have used activityVersionUtil then we should have a ticket to refactor that class in smaller classes based on usage, but we already use methods from there. Thats why in my implementation I used that class